Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'rake'
-gem 'asciidoctor', '1.5.6.1'
+gem 'asciidoctor', '2.0.11'
```

Comment:
asciidoctor 2.x switched to new asciidoc syntax https://asciidoctor.org/docs/migration/#migration-cheatsheet